src/Compilers/Core/Portable/Diagnostic/LocalizableResourceString.cs

```diff
@@ -113,5 +113,23 @@ public override string ToString(IFormatProvider formatProvider)
                 (_formatArguments.Length > 0 ? string.Format(resourceString, _formatArguments) : resourceString) :
                 string.Empty;
         }
+
+        public override bool Equals(LocalizableString other)
```

Comment:
Is the complete description of the API change that they now implement a new interface? The actual semantics of Object.Equals haven't changed nor have any members been removed?